src/lxc/cgroups/cgfsng.c

```diff
@@ -1378,7 +1396,24 @@ static inline bool cgfsng_create(void *hdata)
 	return false;
 }
 
-static bool cgfsng_enter(void *hdata, pid_t pid)
+static inline bool cgfsng_create_inner(struct cgfsng_handler_data *d)
+{
+	size_t i;
+	bool ret = true;
+	char *cgname = must_make_path(d->container_cgroup, "ns", NULL);
```

Comment:
Feels like bike-shedding but is the `ns` supposed to meaningful or just random? Maybe there's a smarter name for this, so it's clear in the `/sys/fs/cgroup/*/*/ns` tree why this cgroup exists. :)